source/Halibut/Transport/Streams/StreamExtensionMethods.cs

```diff
@@ -65,6 +65,17 @@ public static async Task WriteByteArrayAsync(this Stream stream, byte[] bytes, C
             await stream.WriteAsync(bytes, 0, bytes.Length, cancellationToken);
         }
         
+        /// <summary>
+        /// Net48 does not have a method which just writes all bytes of an array to a stream.
+        /// </summary>
+        /// <param name="stream"></param>
+        /// <param name="bytes"></param>
+        /// <param name="cancellationToken"></param>
+        public static void WriteByteArray(this Stream stream, byte[] bytes)
```

Comment:
As we aren't using this in the Halibut itself, I'd be inclined to move it to the test project for now.